### PR TITLE
Add the ability to restrict Bodhi updates using Pagure and query critpath packages from PDC

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

File: bodhi/server/validators.py [hunk 1]
```diff
@@ -329,6 +329,19 @@ def validate_acls(request):
...
+        # but isn't part of the mandatory groups.
+        mandatory_groups = request.registry.settings[
+            'mandatory_packager_groups'].split()
+        for mandatory_group in mandatory_groups:
+            if mandatory_group not in user_groups:
+                error = ("{} is not a member of \"{}\", which is a "
+                         "mandatory packager group").format(
+                    user.name, mandatory_group)
+                request.errors.add('body', 'builds', error)
+                break
```
Comment (ralphbean) [inline bodhi/server/validators.py]: Should this be a `return` instead of a `break`?

File: bodhi/server/util.py [hunk 2]
```diff
@@ -206,6 +206,33 @@ def get_critpath_pkgs(collection='master'):
...
+            '{}?critical_path=true&type=rpm&name={}&active=true&page_size=100'
+            .format(pdc_api_url, collection))
+
+        critpath_pkgs_set = set()
+        while True:
+            pdc_request = requests.get(pdc_api_url_with_args)
+            pdc_request_json = pdc_request.json()
+
+            if pdc_request.status_code != 200:
+                print(pdc_request.status_code)
```
Comment (ralphbean) [inline bodhi/server/util.py]: Drop the `print` and include the status_code in the exception text.

Comment (mprahl) [inline bodhi/server/util.py]: Yes, that was from debugging earlier. Thanks for catching that.

**Comment by mprahl** [comment]
Fixed the changes. Will submit a PR upstream.

**Pull request closed without merging**